src/widgets/splits/SplitHeader.cpp

```diff
@@ -297,6 +298,23 @@ std::unique_ptr<QMenu> SplitHeader::createMainMenu()
 
     moreMenu->addAction(action);
 
+    {
+        auto action = new QAction(this);
+        action->setText("Mute pings");
```

Comment:
```suggestion
        action->setText("Mute highlight sound");
```
In my opinion, using the term "ping" is creating inconsistent copy, since it is not used anywhere else in the UI. The corresponding settings use the terms "Highlights" and "highlight sound".